packages/falso/src/lib/sequence.ts

```diff
@@ -43,11 +41,33 @@ type RandomSequenceOptions = {
  *
  * @example
  *
+ * randSequence({ charType: 'numeric' }) // numeric | alpha | alphaNumeric (can't be used with 'chars' parameter )
```

Comment:
We can remove `(can't be used with 'chars' parameter )` and enforce by using type overloading.

Reply:
If using both doesn't make sense we can enforce it using typescript

Reply:
@NetanelBasal OK, I prepare commit soon

Reply:
@NetanelBasal looks like if I use that code, I really get typescript error, when trying to use wrong method signature, but I lost types because types returned using internal Return<string, Options>, can I expose that type via `export`?